src/Composer/Installers/BaseInstaller.php

```diff
@@ -111,10 +111,29 @@ protected function templatePath($path, array $vars = array())
             }
         }
 
+        if ($this->isAbsolutePath($path) === false) {
+            $path = getcwd() . '/' . $path;
+        }
+
         return $path;
     }
 
     /**
+     * Return true if $path provided is absolute
+     *
+     * @param  string $path
+     * @return boolean 
+     */
+    protected function isAbsolutePath($path)
+    {
+        if ($path === null || $path === '') {
+            return false;
+        }
+
+        return $path[0] === DIRECTORY_SEPARATOR || preg_match('~\A[A-Z]:(?![^/\\\\])~i',$path) > 0;
```

Comment:
You can use Composer\Util\Filesystem->isAbsolutePath() instead of reimplementing this here.